weave-js/src/components/Panel2/PanelGroup.tsx

```diff
@@ -592,9 +606,9 @@ export const PanelGroupConfigComponent: React.FC<PanelGroupProps> = props => {
         </ConfigPanel.ChildConfigContainer>
       </ConfigPanel.ConfigSection>
       <ConfigPanel.ConfigSection>
-        <OldButton size="mini" onClick={handleAddPanel}>
+        <Button className="w-full" variant="secondary" onClick={handleAddPanel}>
           Add Child
-        </OldButton>
+        </Button>
```

Comment:
misc change: replaced old semantic buttons

(first tab is before, second tab is after)

https://github.com/wandb/weave/assets/17016170/dc682319-25c9-470b-8066-bc8a6a0a6a23